recipes/mender-connect/debian-1.2.x

```diff
@@ -0,0 +1 @@
+debian-1.0.x
```

Comment:
Lol, I was staring at this for quite some time before I realized this is a symlink, not the text inside the file.